Improve "Who's Using gVisor" page with semantic HTML and modern, clean markup

NirajDN opened the pull request:
This PR refactors the "Who's Using gVisor" page content to improve readability, accessibility, and maintainability by:

Using semantic HTML5 elements such as section, main ,header and blockquoteto enhance document structure and accessibility.

Simplifying nested markup by removing redundant wrapper <div> elements.

Adding descriptive alt text to images and clear, meaningful link texts to improve SEO and screen reader support.

Applying rel="noopener noreferrer" to external links that open in new tabs to follow security best practices.

Implementing consistent spacing and layout classes to enhance visual clarity and responsiveness.

Adding loading="lazy" to images to optimize page load performance.

Grouping companies into distinct sections with clear headings and content separation.

No changes were made to the actual content — only markup and structural improvements were applied to enhance usability and maintainability.

ayushr2 commented:
Thanks for the PR! Could you upload screenshots showing the change in UI? (I agree that https://gvisor.dev/users/ could use some improvements)

NirajDN commented:
Hi @ayushr2  Thanks for reviewing the PR.

This update is focused on code structure and content cleanup, rather than visual UI changes. Specifically:

Improved semantic HTML and layout consistency

Cleaned up indentation and formatting for better readability

Updated some organization descriptions and link formats

Reduced the overall size of the file from 291 lines to 217 lines (=25% shorter), making it easier to maintain

Since there are no CSS or visual styling changes, the UI looks the same  
so screenshots wouldn’t show meaningful differences.
Thank you for your valuable time.

ayushr2 commented on website/users/index.html:
```diff
@@ -4,288 +4,214 @@
+      </h2>
+      <p>Ant Group develops online payment platforms, offering a wide range of financial services globally.</p>
+      <blockquote class="user-quote p-3 bg-light border rounded">
+        At Ant Group, we are committed to keeping online transactions safe and efficient. Continuously improving security for potential system-level attacks is one of many measures. As a container runtime, gVisor provides container-native security without sacrificing resource efficiency. It has been on our radar since its release.
```
We try to wrap these lines at 80 chars. If you see the older version, all the <p> </p> content was wrapped. Please do the same here.

NirajDN commented:
Hi @ayushr2 ,

Thanks for the note! I've updated the PR to wrap all <p> content at 80 characters, following the formatting used in the original version. Let me know if there's anything else you'd like adjusted.

NirajDN commented:
Hi @ayushr2 ,
I have wrapped the paragraphs to 80 characters as requested, and also squashed the last two commits into one.
Please let me know if there’s anything else you would like me to update.
Thanks again for your guidance!

NirajDN commented:
Hi @ayushr2,
I've wrapped all the requested text blocks to 80 characters. Let me know if there's anything else that needs adjusting. I've also squashed the latest commit as requested.
Thanks!

ayushr2 commented:
I am going to close this PR at this point, it is not making any material changes to the UI and neither are we saving many lines of code. We have already used a lot of time to review this.
